configure.ac

```diff
@@ -717,13 +726,20 @@ AC_SUBST(OLA_CLIENT_LIBS)
 
 # Extra tools
 #####################################################
-AM_CONDITIONAL([BUILD_JA_RULE],
-               [test "x$enable_ja_rule" = xyes && test "x$have_libusb" = xyes \
-                && test "x$have_libusb_hotplug_api" = xyes])
+AS_IF([test "x$enable_ja_rule" = xyes && test "x$enable_usbdmx" != xyes],
+      AC_MSG_ERROR([Ja Rule requires the usbdmx plugin, but it could not be enabled.]))
+AS_IF([test "x$enable_ja_rule" = xyes && test "x$have_libusb" != xyes],
```

Comment:
Do you prefer
    test "x$have_libusb" != xyes
or
    test "x$have_libusb" = xno
? This one is really a boolean so functionally it's the same, but do you have a preference?